src/layout/Header/Header.tsx

```diff
@@ -1,57 +1,63 @@
 import {
-    Avatar,
+    Box,
+    AppBar,
     Button,
     Link,
     Menu,
     MenuItem,
     Typography,
+    IconButton,
+    ListItemIcon,
 } from '@mui/material';
+import { NavLink, useNavigate } from 'react-router-dom';
+import { StyledButton } from '../Footer/Footer.styled';
+import { ConfirmDialog } from '../../components';
+import { StyledToolBar } from './Header.styled';
 import LightModeRoundedIcon from '@mui/icons-material/LightModeRounded';
 import DarkModeRoundedIcon from '@mui/icons-material/DarkModeRounded';
 import DirectionsBusRoundedIcon from '@mui/icons-material/DirectionsBusRounded';
-import testProfile from '../../assets/person1.jpeg';
-import { useState } from 'react';
-import { useAppDispatch, useAppSelector } from '../../app/hooks';
-import { toggleTheme } from '../../app/features/themeSlice';
-import { AuthModal } from '../../components';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import PersonIcon from '@mui/icons-material/Person';
+import ExitToAppIcon from '@mui/icons-material/ExitToApp';
+import PermContactCalendarIcon from '@mui/icons-material/PermContactCalendar';
+import { paths } from '../../config';
 
 export const Header = () => {
-    const { t } = useTranslation('headerFooter'); // mention "ns2" to include values from ns2.json
-    const themeMode = useAppSelector((state) => state.theme.currentTheme);
+    const { t } = useTranslation(['headerFooter', 'logoutModal']);
     const dispatch = useAppDispatch();
+    const navigate = useNavigate();
+
+    const themeMode = useAppSelector((state) => state.theme.currentTheme);
+    const user = useAppSelector((state) => state.auth.user);
+    const [isLogoutModalDisplayed, setIsLogoutModalDisplayed] =
+        useState<boolean>(false);
 
-    const [isLoginClicked, setLoginClick] = useState(false);
-    const [isLoginModalOpen, setIsLoginModalOpen] = useState(false);
     const [menuAnchorElement, setMenuAnchorElement] =
         useState<null | HTMLElement>(null);
 
-    const settings = [t('myBooking'), t('Logout')];
+    const isMenuOpen = Boolean(menuAnchorElement);
 
-    const handleLoginClick = () => {
-        setLoginClick(true);
-        setIsLoginModalOpen(true);
+    const handleOpenUserMenu = (event: React.MouseEvent<HTMLElement>) => {
+        setMenuAnchorElement(event.currentTarget);
     };
 
-    const handleThemeClick = () => {
-        dispatch(toggleTheme());
+    const handleCloseUserMenu = () => {
+        setMenuAnchorElement(null);
     };
 
-    const handleOpenUserMenu = (event: React.MouseEvent<HTMLElement>) => {
-        setMenuAnchorElement(event.currentTarget);
+    const handleThemeClick = () => {
+        dispatch(toggleTheme());
     };
 
-    const handleCloseUserMenu = (setting: string) => {
-        setMenuAnchorElement(null);
-        if (setting === t('Logout')) {
-            setLoginClick(false);
-            setIsLoginModalOpen(false);
-        }
+    const handleLogoutClick = () => {
```

Comment:
Please provide a logout toast to notify the user.

Reply:
Added.